src/axom/slic/interface/slic.hpp

```diff
@@ -25,7 +25,7 @@ namespace slic
 {
 struct debug
 {
-  AXOM_API static bool checksAreErrors;
+  static bool checksAreErrors;
```

Comment:
Likewise, why is `AXOM_API` removed?  Are all symbols exported from Axom by default?